wp-content/mu-plugins/force-plugin-activation.php

```diff
@@ -46,7 +46,6 @@ final class ForcePluginActivation {
 		'advanced-custom-fields-pro/acf.php'                                => [ 'all' ],
 		'block-editor-custom-alignments/block-editor-custom-alignments.php' => [ 'all' ],
 		'core/core.php'                                                     => [ 'all' ],
-		'debug-bar/debug-bar.php'                                           => [ 'local', 'development' ],
```

Comment:
If we're dropping this from force activation, do we still need/want it in the project?